docs/en/04_Changelogs/4.0.0.md

```diff
@@ -1495,19 +1507,27 @@ New `DatetimeField` methods replace `getConfig()` / `setConfig()`:
 
 New `DateField` methods replace `getConfig()` / `setConfig()`:
 
-* `getShowCalendar()` / `setShowCalendar()`
-* `getDateFormat()` / `setShowCalendar()`
+* `getDateFormat()` / `setDateFormat()`
 * `getMinDate()` / `setMinDate()`
 * `getMaxDate()` / `setMaxDate()`
-* `getPlaceholders()` / `setPlaceholders()`
-* `getClientLocale` / `setClientLocale`
 * `getLocale()` / `setLocale()`
-* option `dmyfields` is now superceded with an `SeparatedDateField` class
+
+The `DateField` has changed behavior:
+
+* `DateField` no longer provides a jQuery UI date picker,
+   and uses [HTML5 date pickers](https://www.wufoo.com/html5/types/4-date.html) instead.
+   Use `setUseHTML()` to activate this mode (instead of `setConfig('showcalendar', true)`).
+* `DateField` provides an optional polyfill for
+  [browsers without HTML5 date picker support](http://caniuse.com/#feat=input-datetime) 
+* The `dmyfields` option is now superceded with an `SeparatedDateField` class.
+* `getPlaceholders()` / `setPlaceholders()` moved to a new `SeparatedDateField` class
+* `getClientLocale` / `setClientLocale` have been removed (handled by `DateField->locale` and browser settings)
 
 New `TimeField` methods replace `getConfig()` / `setConfig()`
 
 * `getTimeFormat()` / `setTimeFormat()`
 * `getLocale()` / `setLocale()`
+* `getClientConfig()` has been removed (in favour of `setHTML5()`)
 
 #### <a name="overview-template-removed"></a>Template and Form Removed API
```

Comment:
Need to add notes on the removal of `MemberDatetimeOptionsetField` to this section.